plugin-packages/media/src/audio/audio-loader.ts

```
@@ -0,0 +1,28 @@
+        audioAsset.data = loadedAudios[i] as HTMLAudioElement | AudioBuffer;
+        audioAsset.setInstanceId(audios[i].id);
+        renderer.engine.addInstance(audioAsset);
+      }
+
+      for (let i = 0; i < audios.length; i++) {
+        assetManager.assets[audios[i].id] = loadedAudios[i];
+      }
+    }
+
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)